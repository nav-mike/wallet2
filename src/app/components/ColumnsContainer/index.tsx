import styled from 'styled-components/macro';

export const ColumnsContainer = styled.div`
  display: flex;
  flex-direction: column;
`;
