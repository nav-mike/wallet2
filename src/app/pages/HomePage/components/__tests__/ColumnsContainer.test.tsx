import * as React from 'react';
import { render } from '@testing-library/react';
import { ColumnsContainer } from '../ColumnsContainer';

describe('<ColumnsContainer />', () => {
  it('should be render and match the snapshot', () => {
    const columns = render(<ColumnsContainer />);
    expect(columns.container.firstChild).toMatchSnapshot();
  });
});
