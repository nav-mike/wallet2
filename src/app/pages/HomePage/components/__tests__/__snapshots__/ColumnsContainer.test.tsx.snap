// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ColumnsContainer /> should be render and match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  class="c0"
/>
`;
